Spotify/Spotify.munki.recipe

```diff
@@ -35,15 +35,9 @@ https://github.com/autopkg/recipes/issues/196
             <string>%NAME%</string>
             <key>postinstall_script</key>
             <string>#!/bin/bash
-                # Explicitly setting all binaries to be group and world executable, as the
-                # source dmg may have them set as 744.
-                /bin/chmod go+x \
-                '/Applications/Spotify.app/Contents/Frameworks/Chromium Embedded Framework.framework/Chromium Embedded Framework' \
-                '/Applications/Spotify.app/Contents/Frameworks/Spotify Helper.app/Contents/MacOS/Spotify Helper' \
-                /Applications/Spotify.app/Contents/MacOS/Spotify \
-                /Applications/Spotify.app/Contents/MacOS/SpotifyWebHelper \
-                /Applications/Spotify.app/Contents/MacOS/sp_relauncher \
-                /Applications/Spotify.app/Contents/Library/LoginItems/StartUpHelper.app/Contents/MacOS/StartUpHelper
+                # Add directory traversal for the entire application
```

Comment:
Should be "Add directory traversal for the entire application and ensure all executables are executable by group and other"